packages/embed/src/validation.ts

```diff
@@ -193,6 +218,63 @@ export const validateStore = ({
   return false
 }
 
+const buyerDetails = {
+  firstName: '',
+  lastName: '',
+  emailAddress: '',
+  phoneNumber: '',
+  address: {
+    houseNumberOrName: '',
+    line1: '',
+    line2: '',
+    organization: '',
+    city: '',
+    postalCode: '',
+    country: '',
+    state: '',
+    stateCode: '',
+  },
+  taxId: {
+    value: '',
+    kind: '',
+  },
+}
+
+const buyerObject: Buyer = {
+  billingDetails: buyerDetails,
```

Comment:
Are we missing displayName & externalIdentifier in the buyer object here when validating the schema?

Reply:
yes, updated (also changed the screenshot to show those in the expected schema)